_contrib/maintainers.md

```diff
@@ -0,0 +1,36 @@
+---
+title: The People
+layout: guide
+---
+
+To see what these roles do, and how to become one, look at [governance](./governance.md).
+
+## Triagers
+
+- Eray Karatay (@programatik29)
+- Oddbjørn Grødem (@oddgrd)
+
+## Collaborators
+
+- Sean McArthur (@seanmonstar)
+- Steven Fackler (@sfackler)
+- Oliver Gould (@olix0r)
+- Eliza Weisman (@hawkw)
+- Lucio Franco (@LucioFranco)
+- Anthony Ramine (@nox)
+- David Pedersen (@davidpdrsn)
+- Adam Foltzer (@acfoltzer)
+
+<details>
+<summary>Emeriti</summary>
+
+### Collaborator emeriti
```

Comment:
The markdown in here doesn't apply, I could probably fix this with a CI command but it will add even more complexity.

Reply:
Not a big deal.